tests/ops/qubit/test_observables.py

```diff
@@ -416,16 +416,8 @@ def circuit(basis_state):
             circuit(basis_state)
 
 
-def test_identity_eigvals(tol):
-    """Test identity eigenvalues are correct"""
-    res = qml.Identity._eigvals()
-    expected = np.array([1, 1])
-    assert np.allclose(res, expected, atol=tol, rtol=0)
```

Comment:
how come this test was removed?

Reply:
Since now the identity operator is both an operator and an observable, both qubit and cv, the identity source code is located in its own file in the `ops` folder.  I decided to mirror this structure in the tests folder, making Identity operation tests easier to find.  The identity tests in `test_observables.py` have been moved to `tests/ops/test_identity.py`.

Reply:
Lol I did the same thing in the matrix PR